x/gov/client/utils.go

```diff
@@ -0,0 +1,45 @@
+package client
+
+// NormalizeVoteOption - normalize user specified vote option
```

Comment:
Hmm, I understand the intent but I wonder if it's more obvious to just require exact strings - thoughts @fedekunze ?

Reply:
@faboweb and @jackzampolin suggested that the API should be independent from the backend implementation. So I added the normalization functions.

Reply:
Hmm OK, I don't have a strong opinion